tests/ignite/contrib/metrics/test_cohen_kappa.py

```diff
@@ -224,23 +222,23 @@ def get_test_cases():
 def _test_distrib_integration_binary_input(device):
 
     rank = idist.get_rank()
-    torch.manual_seed(12)
 
     def _test(n_epochs, metric_device):
         metric_device = torch.device(metric_device)
         n_iters = 80
-        s = 16
+        batch_size = 16
         n_classes = 2
```

Comment:
Can you please remove n_classes=2 and put 2 into `y_true = torch.randint(0, n_classes, size=(n_iters * batch_size,)).to(device)` as it is binary data. Thanks